frontend/src/components/eam-configurate-group/eam-configurate-group.module.scss

```diff
@@ -64,10 +64,6 @@
   max-width: 50%;
 }
 
-.table {
-  padding: 0;
-}
```

Comment:
why?
attache the screenshot how it looks like

Reply:
I put this style in a folder for each table